### PR TITLE
Vagrant bump

### PR DESCRIPTION
**Pull request by gyakovlev**
@hydrapolic hey!
tested logging into cloud, firing up and ssh-ing into machines. no problems.

```diff
diff -u vagrant-2.1.5.ebuild vagrant-2.2.1.ebuild 
--- vagrant-2.1.5.ebuild        2018-10-21 14:15:26.271608606 -0700
+++ vagrant-2.2.1.ebuild        2018-11-15 21:54:59.731852744 -0800
@@ -1,7 +1,7 @@
 # Copyright 1999-2018 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="6"
+EAPI=6
 USE_RUBY="ruby23 ruby24 ruby25"
 
 RUBY_FAKEGEM_EXTRADOC="CHANGELOG.md README.md"
@@ -27,25 +27,30 @@
        virtualbox? ( || ( app-emulation/virtualbox app-emulation/virtualbox-bin ) )"
 
 ruby_add_rdepend "
+       >=dev-ruby/bcrypt_pbkdf-1.0.0
        >=dev-ruby/childprocess-0.6.0
+       >=dev-ruby/ed25519-1.2.4
        >=dev-ruby/erubis-2.7.0
-       <dev-ruby/i18n-0.8.0:*
-       >=dev-ruby/listen-3.1.5
        >=dev-ruby/hashicorp-checkpoint-0.1.5
+       >=dev-ruby/i18n-1.1.1:*
+       >=dev-ruby/listen-3.1.5
        >=dev-ruby/log4r-1.1.9 <dev-ruby/log4r-1.1.11
+       <dev-ruby/mime-types-3:*
        >=dev-ruby/net-ssh-5.0.0:*
        >=dev-ruby/net-sftp-2.1
        >=dev-ruby/net-scp-1.2.0
        || ( dev-ruby/rest-client:2 >=dev-ruby/rest-client-1.6.0:0 )
-       <dev-ruby/mime-types-3:*
+       >=dev-ruby/rubyzip-1.2.2
+       >=dev-ruby/vagrant_cloud-2.0.0
 "
+#
 
 # upstream specifies rake>=12 but it apparently doesn't need something this
 # recent. Because vagrant builds fine with rake 10 and because stabilizing rake
 # is tricky, we specify a lower dependency requirement here. This way, we'll be
 # able to stabilize vagrant sooner.
 ruby_add_bdepend "
-       >=dev-ruby/rake-10.0.0
+       >=dev-ruby/rake-10.5.0
 "
 
 all_ruby_prepare() {
@@ -70,13 +75,17 @@
 }
 
 all_ruby_install() {
-       newbashcomp contrib/bash/completion.sh ${PN}
        all_fakegem_install
 
+       newbashcomp contrib/bash/completion.sh ${PN}
+
        # provide executable similar to upstream:
        # https://github.com/hashicorp/vagrant-installers/blob/master/substrate/modules/vagrant_installer/templates/vagrant.erb
        dobin "${PN}"
 
        # directory for plugins.json
        keepdir /var/lib/vagrant
+
+       insinto /usr/share/vim/vimfiles/syntax/
+       doins contrib/vim/vagrantfile.vim
 }
```

@graaff hey, can you please check out vagrant_cloud ebuild once you have a chance? It's my first ruby ebuild written from scratch. tests pass, same output as from upstream travis but with rubocop wiped out. on vagrant side everything works.

### CODE REVIEW

File: dev-ruby/vagrant_cloud/vagrant_cloud-2.0.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+USE_RUBY="ruby23 ruby24 ruby25"
+
+RUBY_FAKEGEM_EXTRADOC="CHANGELOG.md README.md"
+RUBY_FAKEGEM_GEMSPEC="${PN}.gemspec"
+RUBY_FAKEGEM_RECIPE_TEST="rspec3"
+RUBY_FAKEGEM_TASK_DOC=""
```
Comment (hydrapolic) [inline dev-ruby/vagrant_cloud/vagrant_cloud-2.0.1.ebuild]: We can probably remove this.

File: dev-ruby/vagrant_cloud/vagrant_cloud-2.0.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+inherit ruby-fakegem
+
+DESCRIPTION="Vagrant Cloud API Library"
+HOMEPAGE="https://github.com/hashicorp/vagrant_cloud"
+SRC_URI="https://github.com/hashicorp/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
```
Comment (hydrapolic) [inline dev-ruby/vagrant_cloud/vagrant_cloud-2.0.1.ebuild]: This too.

File: dev-ruby/vagrant_cloud/vagrant_cloud-2.0.1.ebuild (hunk 1 above)
Comment (gyakovlev) [reply dev-ruby/vagrant_cloud/vagrant_cloud-2.0.1.ebuild]: nope, removing it adds `doc` useflag, but I haven't tested what happens If I actually enable it =D
probably nothing good.

File: dev-ruby/vagrant_cloud/vagrant_cloud-2.0.1.ebuild (hunk 2 above)
Comment (gyakovlev) [reply dev-ruby/vagrant_cloud/vagrant_cloud-2.0.1.ebuild]: I like to keep those, just preference, 15439 other ebuilds do the same.

File: dev-ruby/vagrant_cloud/vagrant_cloud-2.0.1.ebuild (hunk 1 above)
Comment (gyakovlev) [reply dev-ruby/vagrant_cloud/vagrant_cloud-2.0.1.ebuild]:
```
rake aborted!
LoadError: cannot load such file -- bundler/gem_tasks
/tmp/portage/dev-ruby/vagrant_cloud-2.0.1/work/all/vagrant_cloud-2.0.1/Rakefile:3:in `<top (required)>'
(See full trace by running task with --trace)
 * ERROR: dev-ruby/vagrant_cloud-2.0.1::gentoo failed (compile phase):
 *   failed to (re)build documentation
```

yep, looks like there is no docs there.

**Comment by hydrapolic** [comment]
As discussed on irc, it seems to fail for some custom images while connecting, but some others work fine, I'm for merging this but keeping 2.1.5. Thanks for your work on this @gyakovlev :)

**Comment by gyakovlev** [comment]
looks like there is ver 6 of virtualbox coming soon and vagrant changes are partially related to this.
going to test it before merge, probably we'll want to adjust older ebuilds to depend on <=virtualbox-5.*, but I haven't looked close yet, just guessing. will look at this closer this weekend.

**Comment by hydrapolic** [comment]
Ack for dropping myself as maintainer, thanks @gyakovlev.

**Pull request closed without merging**